databricks/koalas/series.py

```diff
@@ -258,6 +258,36 @@ def spark_type(self):
         return self.schema.fields[-1].dataType
 
     def astype(self, dtype):
+        """
+        Cast a Koalas object to a specified dtype ``dtype``.
+
+        Parameters
+        ----------
+        dtype : data type
+            Use a numpy.dtype or Python type to cast entire pandas object to
+            the same type.
+
+        Returns
+        -------
+        casted : same type as caller
+
+        See Also
+        --------
+        to_datetime : Convert argument to datetime.
+        numpy.ndarray.astype : Cast a numpy array to a specified type.
+
+        Examples
+        --------
+        >>> ser = ks.Series([1, 2], dtype='int32')
+        >>> ser
+        0    1
+        1    2
+        Name: 0, dtype: int32
+        >>> ser.astype('int64')
```

Comment:
add a new line here ...

in general i think we should separate doctests with blank lines to make them easier to read

Reply:
okie